pennylane/ops/qubit/matrix_ops.py

```diff
@@ -176,7 +176,8 @@ def pow(self, z):
         return super().pow(z)
 
     def _controlled(self, wire):
-        ControlledQubitUnitary(*self.parameters, control_wires=wire, wires=self.wires)
+        new_op = ControlledQubitUnitary(*self.parameters, control_wires=wire, wires=self.wires)
+        return new_op.inv() if self.inverse else new_op
```

Comment:
`ControlledOperation` just treated `_controlled` as a quantum function that records the operation into a tape.  I do all this without queueing.

Also, previously, it would have ignored in-place inversion.

Reply:
?

Reply:
I'm wondering, as you change `_controlled` in many operations here, would users have to do the same with their custom operations? And if they don't, does that simply mean that their operations do not work with `Controlled`, or does `ControlledOperation` break?

Reply:
It's an undocumented private method.  They use it at their own risk and know there could be breaking changes without warning.

Reply:
Fair enough! :)